### PR TITLE
Add API for get user org permissions

### LINKED ISSUE #17173: [API] Get org permissions
### Feature Description

It would be very nice if there is an endpoint to get a summary of the permissions a user has in an org (if the user is in a team that has access to the certain permissions):
1. Admin status (if user is an org admin/owner)
2. Repository creation

If the user isn't in the org it could just return "false" for both fields or just return an error.

### Screenshots

_No response_

### PR DESCRIPTION
**Pull request by romdum**
Closes #17173

Add new API endpoint to get user permissions on an organization.

**URL**: `/api/v1/users/{user}/orgs/{org}/permissions`

**Method**: GET

**Response example**: 
```json
{
    "is_owner": false,
    "is_admin": true,
    "can_write": true,
    "can_read": true,
    "can_create_repository": false
}
```

### CODE REVIEW

**Comment by 6543** [comment]
```
could not resolve reference in /users/{username}/orgs/{org}/permissions to $ref #/responses/OrganizationPermissions: object has no key "OrganizationPermissions"
```

**Comment by 6543** [comment]
just read https://github.com/go-gitea/gitea/blob/main/CONTRIBUTING.md#api-v1 and you will find the answer ;)

**Comment by romdum** [comment]
> just read https://github.com/go-gitea/gitea/blob/main/CONTRIBUTING.md#api-v1 and you will find the answer ;)

I just push a fix with the swaggerResponse, thanks for helping :)

**Comment by 6543** [comment]
Ok one thing missing: cam you add some api integration **tests** ... ther should be enout fixtures so just query 3-4 users for 1-2 orgs ...

Example: https://github.com/go-gitea/gitea/blob/main/integrations/api_notification_test.go

And after that we are good to go :)

**Comment by 6543** [comment]
Fixtures: https://github.com/go-gitea/gitea/tree/main/models/fixtures

(*Dummy DB* to test against ..)

File: routers/api/v1/org/org.go [hunk 1]
```diff
@@ -97,6 +97,76 @@ func ListUserOrgs(ctx *context.APIContext) {
...
+	//   description: username of user
+	//   type: string
+	//   required: true
+	// - name: org
+	//   in: path
+	//   description: name of the organization
+	//   type: string
+	//   required: true
+	// responses:
+	//   "200":
```
Comment (6543) [inline routers/api/v1/org/org.go:119]: add resp 404 desc please :)

if you need a hint look into other swagger desc

**Review by 6543 — APPROVED** [review]

File: routers/api/v1/org/org.go [hunk 2]
```diff
@@ -97,6 +97,80 @@ func ListUserOrgs(ctx *context.APIContext) {
...
+
+	authorizeLevel, err := o.GetOrgUserMaxAuthorizeLevel(u.ID)
+	if err != nil {
+		ctx.Error(http.StatusInternalServerError, "GetOrgUserAuthorizeLevel", err)
+		return
+	}
+
+	op.IsOwner = false
+	op.IsAdmin = false
+	op.CanWrite = false
```
Comment (wxiaoguang) [inline routers/api/v1/org/org.go]: Why `IsOwner` `IsAdmin` `CanWrite` are set to false, but others like `CanRead` `CanCreateRepository` are not touched?

All boolean variables should be default to false?

Comment (6543) [reply routers/api/v1/org/org.go]: heh right :)

Comment (6543) [inline routers/api/v1/org/org.go]:
```suggestion
```

**Review by wxiaoguang — APPROVED** [review]

**Pull request merged**